Multichain-proposal.md

```diff
@@ -0,0 +1,112 @@
+# Multichain Proposal for Cardano
+
+# Introduction to Multichain
+
+Multichain is the ultimate Router for web3. It is an infrastructure developed for arbitrary cross-chain interactions.
+
+Multichain was born as Anyswap on the 20th July 2020 to service the clear needs of different and diverse blockchains to communicate with each other. Each blockchain has its own unique services that it provides, its own community and its own development ecosystem. For our industry to reach the next level for consumers, we need a fast, secure, inexpensive and reliable way to exchange value, data and exercise control between the chains.
+
+The solutions developed by Multichain allow almost all blockchains to inter-operate. There is no restriction to Ethereum like chains (e.g. Binance Smart Chain), or different Layer 2 chains requiring finality to Ethereum (e.g. Polygon), or a network of Parachains (e.g. Moonbeam in the PolkaDot system), or Bitcoin types of chain (e.g. Litecoin), or COSMOS chains (e.g. Terra). These are either now all integrated, or on course for integration. With support for all ECDSA and EdDSA encrypted chains, Multichain is almost universally applicable as an interoperable layer.
```

Comment:
Could you share some documentation/specification of Litecoin bridging? Probably it's useful to link it given the model similarities.

Reply:
You can check out the BTC repo https://github.com/anyswap/CrossChain-Router/tree/feature/btc